src/app/app-testing/app-testing.component.css

```diff
@@ -0,0 +1,4 @@
+p {
+    color: red;
```

Comment:
We should add colors to a variables file instead

Reply:
Understood